ffi/dart/test/icu_test.dart

```diff
@@ -145,21 +167,14 @@ void main() {
     );
 
     expect(
-      ZonedDateTimeFormatter.withLength(locale, DateTimeLength.long).formatIso(
-        zonedDateTimeIso.date,
-        zonedDateTimeIso.time,
-        zonedDateTimeIso.zone,
-      ),
-      '14. Radschab 1446 AH, 14:32:12 MEZ',
-    );
-
-    expect(
-      ZonedDateTimeFormatter.withLength(locale, DateTimeLength.short).formatIso(
-        zonedDateTimeIso.date,
-        zonedDateTimeIso.time,
-        zonedDateTimeIso.zone,
-      ),
-      '14.07.46 AH, 14:32:12 MEZ',
+      DateTimeFormatterGregorian.ymdet(locale)
+          .withZoneGenericLong(locale)
+          .formatIso(
+            zonedDateTimeIso.date,
+            zonedDateTimeIso.time,
+            zonedDateTimeIso.zone,
+          ),
+      'Mi., 15.01.2025, 14:32:12 Mitteleuropäische Zeit',
```

Comment:
can you produce the same output as before?

Reply:
Done